File: Stage_0_Christabel.R
name <- 'Mbachu Christabel'
email <- "mbachuanna@gmail.com"
language <- 'R'
biostack <- "Genomics"
slackuser <- "@Christabel"
twitter <- "@kristahbel"
distance <- sum(a != b)
data <- data.frame(Details=c(name, email, language, biostack, slackuser, twitter, distance))
rownames(data) <- c('NAME: ', 'E-MAIL: ', 'LANGUAGE: ', 'BIOSTACK: ', 'SLACK USERNAME: ', 'TWITTER: ', 'DISTANCE: ')
colnames <- NULL
a = unlist(strsplit(slackuser, ""))
b = unlist(strsplit(twitter, ""))
sum(a != b)

print(data)

File: stage_0_Christabel.py
print("NAME: Mbachu Christabel \nE-MAIL: mbachuanna@gmail.com \nSLACK USERNAME: @Christabel \nBIOSTACK: Genomics \nTwitter Handle: @kristahbel") 
def hamming_distance(a,b):
    count=0
    for i in range(len(a)):
        if a[i] != b[i]:
            count +=1
    return count

print(hamming_distance('@Christabel','@kristahbel')) 
